lib/models/device.coffee

```diff
@@ -249,6 +249,41 @@ exports.getApplicationName = (uuid, callback) ->
 	exports.get(uuid).get('application_name').nodeify(callback)
 
 ###*
+# @summary Get application container information
+# @name getApplicationInfo
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {Object} - application info
+# @returns {Promise}
+#
+# @example
+# resin.models.device.getApplicationInfo('7cf02a6').then(function(appInfo) {
+# 	console.log(appInfo);
+# });
+#
+# @example
+# resin.models.device.getApplicationInfo('7cf02a6', function(error, appInfo) {
+# 	if (error) throw error;
+# 	console.log(appInfo);
+# });
+###
+exports.getApplicationInfo = (uuid, callback) ->
+	exports.get(uuid).then (device) ->
+		appId = device.application[0].id
+		return request.send
+			method: 'GET'
+			url: "/supervisor/v1/apps/#{appId}"
+			baseUrl: settings.get('apiUrl')
+			body:
```

Comment:
Done, added `uuid: uuid` instead to be consistent with the `/blink` endpoint request.